demo/app.component.ts

```diff
@@ -394,6 +404,31 @@ export class AppComponent implements OnInit {
     AppComponent.logEvent(e, 'Collapsed');
   }
 
+  public selectNodeFromFFS(id: number | string) {
+    let treeController = this.treeFFS.getChildControllerById(id);
+    treeController.select();
+  }
+
+  public switchFFS(id: number | string) {
+    let treeController = this.treeFFS.getChildControllerById(id);
+    treeController.switchFoldingType();
```

Comment:
@lovery Do we really need to expose this? Isn't enough to have just `expand` and `collpase`?